docs_nnx/guides/bridge_guide.md

```diff
@@ -29,72 +47,91 @@ from jax.experimental import mesh_utils
 
-This means you can move in either top-down or bottom-up behavior: convert the whole Linen module to NNX, then gradually move down, or convert all the lower level modules to NNX then move up.
+Therefore, you can convert the entire `flax.linen.Module` to Flax NNX, and then gradually “move down” (the “top-down” way), or convert all the lower-level `flax.linen.Module`s to Flax NNX and then “move up” (the “bottom-up” way).
 
++++
 
-## The Basics
+## Basics
 
-There are two fundamental difference between Linen and NNX modules:
+There are two fundamental differences between [`flax.linen.Module`](https://flax-linen.readthedocs.io/en/latest/api_reference/flax.linen/module.html#flax.linen.Module) and [`flax.nnx.Module`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/module.html#flax.nnx.Module):
 
-* **Stateless vs. stateful**: Linen module instances are stateless: variables are returned from a purely functional `.init()` call and managed separately. NNX modules, however, owns its variables as instance attributes.
+* **Stateless vs stateful**:
+  - Flax Linen `Module` instances are stateless: Variables are returned from a purely functional `Module.init()` call and managed separately.
+  - Flax NNX `Module`s, however, own their variables as instance attributes.
 
-* **Lazy vs. eager**: Linen modules only allocate space to create variables when they actually see their input. Whereas NNX module instances create their variables the moment they are instantiated, without seeing a sample input.
+* **Lazy vs eager**:
+  - Flax Linen `Module`s only allocate space to create variables when they actually see their input.
+  - In comparison, Flax NNX `Module` instances create their [`nnx.Variable`s](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/variables.html#flax.nnx.Variable) the moment they are instantiated without seeing a sample input.
 
-With that in mind, let's look at how the `nnx.bridge` wrappers tackle the differences.
+With that in mind, let's investigate how the [`nnx.bridge`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/bridge.html) wrappers tackle these differences.
 
-### Linen -> NNX
+### Basics: Flax Linen to NNX with `nnx.bridge.lazy_init`
 
-Since Linen modules may require an input to create variables, we semi-formally supported lazy initialization in the NNX modules converted from Linen. The Linen variables are created when you give it a sample input.
+Since `flax.linen.Module`s may require an input to create variables, the Flax team semi-formally supports lazy initialization in the `flax.nnx.Module`s converted from Flax Linen. The Flax Linen variables are created when you give it a sample input. For you, it's calling [`nnx.bridge.lazy_init`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/bridge.html#flax.nnx.bridge.ToNNX.lazy_init) (`nnx.bridge.ToNNX.lazy_init`) where you call `module.init()` in the Flax Linen code.
```

Comment:
@cgarciae We should mention the `ToNNX` bridge here since it's used in the code snippet below.